runway/cfngin/exceptions.py

```diff
@@ -602,21 +747,32 @@ def __init__(self, blueprint_name: str, *args: Any, **kwargs: Any) -> None:
             **kwargs: Arbitrary keyword arguments.
 
         """
+        self.args = args
+        self.kwargs = kwargs
```

Comment:
`args` & `kwargs` don't need to be saved since they arn't used (other than to pass to `super()`).